.github/actions/spell-check/expect/expect.txt

```diff
@@ -2828,4 +2828,4 @@ zy
 AAAAABBBBBBCCC
 AAAAA
 BBBBBCCC
-abcd
\ No newline at end of file
+abcd
```

Comment:
huh, another change that isn't really a change...

Reply:
its a whitespace EOL. ignore it. it was warning in the spell checker.